spec/requests/api/moves_controller_update_spec.rb

```diff
@@ -400,7 +400,7 @@
           end
 
           context 'when the supplier has a webhook subscription', :skip_before do
-            let!(:subscription) { create(:subscription, :no_email_address, supplier: supplier) }
+            let!(:subscription) { create(:subscription, :no_email_address, supplier: move.supplier) }
```

Comment:
Presumably this spec file defines a `supplier` let somewhere - better instead of this change to use that supplier when creating the move instead; otherwise this ends up creating two suppliers which is slower.

Reply:
Ditto other amended specs in this file.

Reply:
specifying the supplier, the subscription factory does not create its own supplier, so it reduces the test of -1 supplier

Reply:
Fixed.